src/Pair.sol

```diff
@@ -16,6 +16,9 @@ abstract contract Pair is IPair, UniswapV2ERC20 {
 
     bytes4 private constant SELECTOR        = bytes4(keccak256("transfer(address,uint256)"));
     uint256 public constant MINIMUM_LIQUIDITY  = 10**3;
+    string private constant PLATFORM_FEE_NAME = "Shared::platformFee";
+    string private constant RECOVERER_NAME = "Shared::defaultRecoverer";
+    string internal constant PLATFORM_FEE_TO_NAME = "Shared::platformFeeTo";
 
```

Comment:
The alignment looks a little drunk here :P

Reply:
rearranged and re-aligned them. Should be a lot better now 😃